files/style.css

```diff
@@ -0,0 +1,175 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #1d2379;
+  position: absolute;
+  background-image: url("Ellipse 18.png");
+  background-repeat: no-repeat;
+}
+
+.heading {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
- Please use a similar social media icon to the Figma design as yours is different from the Figma design this is to help you follow the details of a design properly. And also change the color of the horizontal line to be similar to that of the Figma design. `background: #646369;`

##### Your design 
![Screenshot from 2022-07-01 19-29-00](https://user-images.githubusercontent.com/58771507/176951552-d934b998-d87d-4081-b041-504c4e1ed983.png)


##### Figma design 
![Screenshot from 2022-07-01 19-30-02](https://user-images.githubusercontent.com/58771507/176951658-d8aa76b2-6178-4241-a571-1e3c0456ae7d.png)